msteams-platform/concepts/build-and-test/deep-links-execution-handling.md

```diff
@@ -0,0 +1,107 @@
+
+```javascript
+if (pages.isSupported()) {
+  const navPromise = pages.navigateToApp({ appId: <appId>, pageId: <pageId>, webUrl: <webUrl>, subPageId: <subPageId>, channelId:<channelId>});
+  navPromise.
+     then((result) => {/*Successful navigation*/}).
+     catch((error) => {/*Failed navigation*/});
+}
+else { /* handle case where capability isn't supported */ }
+```
+
+For more information about using the pages capability, see [pages.navigateToApp()](/javascript/api/@microsoft/teams-js/pages?view=msteams-client-js-latest#@microsoft-teams-js-pages-navigatetoapp&preserve-view=true) and the [pages](/javascript/api/@microsoft/teams-js/pages?view=msteams-client-js-latest&preserve-view=true) namespace for other navigation options. If needed, it's also possible to directly open a deep link using the [app.openLink()](/javascript/api/@microsoft/teams-js/app?view=msteams-client-js-latest#@microsoft-teams-js-app-openlink&preserve-view=true) function.
+
+# [TeamsJS v1](#tab/teamsjs-v1)
+
+To trigger a deep link from your tab, call:
+
+```javascript
+microsoftTeams.executeDeepLink(/*deepLink*/);
+```
+
+---
+
+Applications can also use the SDK to execute certain scenarios without building a deep link themselves. Refer to each scenario for sample code.
+
+## Chat message
+
+Applications can post messages to a chat and add deep links to them. The following are different ways an application can include a deep link in a message.
+
+### Hyperlink text message
+
+Applications can add a deep link as a hyperlink markdown. The deep link is executed within teams. Refer the following code sample::
+
+`add sample`
+
+### Raw link text message
+
+Applications can include the raw link in a message. The deep link is executed within teams. Refer the code below:
+
+`add sample`
+
+### Adaptive Card – Open URL action  
+
+Applications can include an `open URL` action in an Adaptive Card and add a deep link to it. The deep link is executed within Teams. Refer to the following sample Adaptive Card payload:
+
+`add sample`
+
+### Adaptive Card as hyperlink Markdown (Not recommended)
+
+Applications can include a hyperlink Markdown in an Adaptive Card. This opens in browser first. Refer to the following sample payload:
+
+`add sample`
+
+### Adaptive Card as raw link (Not recommended)
+
+Raw link in Adaptive Card body doesn't have any action associated.
+
+## Bot
+
+### Open URL action in bot
+
+Applications can include an `open URL` action in a bot and can add a deep link to it. The deep link is executed within Teams. Refer to the following sample Adaptive Card payload:
+
+`add sample`
+
+### Card body of the bot
+
+Applications can include the raw link in a Card body. The deep link opens in a browser first. Refer the following code sample:
+
+`add sample`
+
+#### Hyperlink Markdown in a Bot
+
+Applications can include a hyperlink Markdown in a Bot. This opens in a browser first. Refer to sample payload below:
+
+`add sample`
```

Comment:
Remove the BOT section. We have already covered messages in the previous section. (raw text message and adaptive cards)

Reply:
Removed bots section